ui/raidboss/data/02-arr/dungeon/brayfloxs_longstop.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+[{
+  zoneId: ZoneId.BrayfloxsLongstop,
+  triggers: [
+    {
+      // Numbing Breath is Stunnable
+      id: 'Brayflox Normal Numbing Breath',
+      netRegex: NetRegexes.startsUsing({ id: '1FA', source: 'Great Yellow Pelican' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Pelican Adds
+      id: 'Brayflox Normal Pelican Adds',
+      netRegex: NetRegexes.message({ line: 'A flock of ziz emerges from the jungle!', capture: false }),
+      response: Responses.killAdds('info'),
+    },
+    {
+      // Ashdrake Cyclone Stun
+      id: 'Brayflox Normal Ashdrake Burning Cyclone',
+      netRegex: NetRegexes.startsUsing({ id: '205', source: 'Ashdrake' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Tempest Biast Spawn
+      id: 'Brayflox Normal Tempest Biast',
+      netRegex: NetRegexes.addedCombatantFull({ npcNameId: '1285', capture: false }),
+      response: Responses.killAdds('info'),
+    },
+    {
+      // Inferno Drake Cyclone Stun
+      id: 'Brayflox Normal Inferno Drake Burning Cyclone',
+      netRegex: NetRegexes.startsUsing({ id: '3D8', source: 'Inferno Drake' }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      response: Responses.stun('info'),
+    },
+    {
+      // Hellbender Bubble Others
+      id: 'Brayflox Normal Hellbender Effluvium Others',
+      netRegex: NetRegexes.startsUsing({ id: '3D3', source: 'Hellbender' }),
+      condition: function(data, matches) {
+        return matches.target != data.me;
+        },
+      alertText: function(data, matches) {
+        return {
+          en: 'Break Bubble on ' + data.ShortName(matches.target),
+        };
+      },
+    },
+    {
+      // Hellbender Bubble You
+      id: 'Brayflox Normal Hellbender Effluvium You',
+      netRegex: NetRegexes.startsUsing({ id: '3D3', source: 'Hellbender' }),
+      condition: function(data, matches) {
+        return matches.target == data.me;
+        },
+      alertText: function(data, matches) {
+        return {
+          en: 'Break Bubble on YOU'

```

Comment:
I realize this is the way a lot of other triggers are written, but probably "Break Your Bubble" reads better.  I think of this kind of thing as "Mechanic on YOU" and I don't think "Break Bubble" is the mechanic, if that makes sense.